static/js/amp_store.js

```diff
@@ -0,0 +1,49 @@
+window.onload = function(){
```

Comment:
since we are comparing objects within the same type, should consider `===` and `!==` in if statements, see [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness)

Reply:
updated